DataProcessing/datax-utility/pom.xml

```diff
@@ -164,132 +164,79 @@ SOFTWARE
         </dependency>
     </dependencies>
 
-  <profiles>
-    <profile>
-      <id>build</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
-      <build>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <!-- Build uber jar -->
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-shade-plugin</artifactId>
+                    <version>3.2.1</version>
```

Comment:
Since we currently using 3.2.2 version of maven-shade-plugin, why is the need to downgrade to 3.2.1